app/addons/databases/actions.js

```diff
@@ -155,7 +155,7 @@ export default {
           type: 'success',
           clear: true
         });
-        var route = FauxtonAPI.urls('allDocs', 'app', app.utils.safeURLName(databaseName), '?limit=' + Resources.DocLimit);
+        var route = FauxtonAPI.urls('allDocs', 'app', encodeURIComponent(databaseName), '?limit=' + Resources.DocLimit);
```

Comment:
Could you rather use this function https://github.com/apache/couchdb-fauxton/blob/master/app/addons/documents/base.js#L40 and then you don't need to encodeURIComponent